resources/js/plugins/darkmode/index.js

```diff
@@ -1,6 +1,11 @@
 export default {
     install(vue) {
         vue.mixin({
+            created() {
+                if (this.currentTheme === null) {
+                    localStorage.setItem('theme', this.currentTheme = 'light');
```

Comment:
Suspect `this.currentTheme = 'light'` will cause errors too and needs to be defined separately on the line above